### DISCUSSION TITLE
Vivobook 14

### DISCUSSION THREAD

**Discussion by Topbanaana** [SpiralLinux/SpiralLinux-project#128] [General]
This is out of discussion.

SpiralLinux is an excellent, shining, top performing distro for my decent hardware.

toppy@vivo:~ $ inxi -F
System:    Host: vivo Kernel: 5.19.9-xanmod1-x64v2 x86_64 bits: 64 Desktop: Budgie 10.5.2 
           Distro: Debian GNU/Linux 11 (bullseye) 
Machine:   Type: Laptop System: ASUSTeK product: Vivobook_ASUSLaptop M3401QC v: 1.0 serial: <superuser required> 
           Mobo: ASUSTeK model: M3401QC v: 1.0 serial: <superuser required> UEFI: American Megatrends LLC. v: M3401QC.305 
           date: 03/07/2022 
Battery:   ID-1: BAT0 charge: 58.3 Wh condition: 58.3/63.0 Wh (93%) 
CPU:       Info: 8-Core model: AMD Ryzen 7 5800H with Radeon Graphics bits: 64 type: MT MCP L2 cache: 4 MiB 
           Speed: 4463 MHz min/max: 400/4463 MHz Core speeds (MHz): 1: 4463 2: 684 3: 400 4: 400 5: 3543 6: 400 7: 1091 8: 400 
           9: 2535 10: 400 11: 441 12: 400 13: 400 14: 400 15: 4463 16: 400 
Graphics:  Device-1: NVIDIA GA107 driver: nouveau v: kernel 
           Device-2: Advanced Micro Devices [AMD/ATI] Cezanne driver: amdgpu v: kernel 
           Device-3: IMC Networks USB2.0 HD UVC WebCam type: USB driver: uvcvideo 
           Display: x11 server: X.Org 1.20.11 driver: loaded: amdgpu,ati,modesetting unloaded: fbdev,vesa 
           resolution: 2880x1800~ 90Hz 
           OpenGL: renderer: AMD RENOIR (DRM 3.47.0 5.19.9-xanmod1-x64v2 LLVM 11.0.1) v: 4.6 Mesa 20.3.5 
Audio:     Device-1: Advanced Micro Devices [AMD/ATI] driver: snd_hda_intel 
           Device-2: Advanced Micro Devices [AMD] Raven/Raven2/FireFlight/Renoir Audio Processor driver: N/A 
           Device-3: Advanced Micro Devices [AMD] Family 17h HD Audio driver: snd_hda_intel 
           Sound Server: ALSA v: k5.19.9-xanmod1-x64v2 
Network:   Device-1: MEDIATEK driver: mt7921e 
           IF: wlan0 state: up mac: 48:e7:da:ba:95:49 
           Device-2: Realtek RTL8153 Gigabit Ethernet Adapter type: USB driver: r8152 
           IF: enx00e04c680820 state: up speed: 1000 Mbps duplex: full mac: 01:e0:4c:68:08:27 
Bluetooth: Device-1: IMC Networks Wireless_Device type: USB driver: btusb 
           Report: ID: hci0 state: up running pscan bt-v: 3.0 address: 49:E7:DA:BA:95:47 
Drives:    Local Storage: total: 3.22 TiB used: 963.27 GiB (29.2%) 
           ID-1: /dev/nvme0n1 vendor: Kingston model: SNVS2000G size: 1.82 TiB 
           ID-2: /dev/sda type: USB model: Multiple Card Reader size: 58.24 GiB 
           ID-3: /dev/sdb type: USB vendor: Western Digital model: WDS480G2G0B-00EPW0 size: 447.14 GiB 
           ID-4: /dev/sdc type: USB vendor: Western Digital model: WDS1 00T2B0A-00SM50 size: 931.51 GiB 
Partition: ID-1: / size: 1.82 TiB used: 351.74 GiB (18.9%) fs: btrfs dev: /dev/nvme0n1p2 
           ID-2: /boot/efi size: 299.4 MiB used: 3.2 MiB (1.1%) fs: vfat dev: /dev/nvme0n1p1 
           ID-3: /home size: 1.82 TiB used: 351.74 GiB (18.9%) fs: btrfs dev: /dev/nvme0n1p2 
           ID-4: /tmp size: 1.82 TiB used: 351.74 GiB (18.9%) fs: btrfs dev: /dev/nvme0n1p2 
           ID-5: /var/log size: 1.82 TiB used: 351.74 GiB (18.9%) fs: btrfs dev: /dev/nvme0n1p2 
Swap:      ID-1: swap-1 type: zram size: 14.29 GiB used: 0 KiB (0.0%) dev: /dev/zram0 
Sensors:   System Temperatures: cpu: 46.0 C mobo: N/A gpu: amdgpu temp: 40.0 C 
           Fan Speeds (RPM): cpu: 2400 
Info:      Processes: 416 Uptime: 1h 21m Memory: 15.04 GiB used: 3.65 GiB (24.2%) Shell: Bash inxi: 3.3.01 
toppy@vivo:~$ 

Kudos to all developers.

**Comment by geckolinux** [comment]
Hi there, really appreciate the success report! Thrilled to hear that it's all working well. From the MAC address and given the Vivobook model lineup it looks like it has Realtek WiFI? Any idea which specific WiFI chipset it is, and how reliable and performant is the connection on SpiralLinux? That was a big goal of mine to support Realtek WiFI, which is normally a huge pain in Linux, but I don't have any hardware to test it.
